clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -93,6 +93,17 @@ public class TopicConfig {
             "deletes the old segments. Default value is -2, it represents `retention.bytes` value to be used. The effective value should always be " +
             "less than or equal to `retention.bytes` value.";
 
+    public static final String REMOTE_LOG_DISABLE_POLICY_RETAIN = "retain";
+    public static final String REMOTE_LOG_DISABLE_POLICY_DELETE = "delete";
+
+    public static final String REMOTE_LOG_DISABLE_POLICY_CONFIG = "remote.log.disable.policy";
+
+    public static final String REMOTE_LOG_DISABLE_POLICY_DOC = String.format("Determines whether tiered data for a topic should be retained or " +
+            "deleted after tiered storage disablement on a topic. The two valid options are \"%s\" and \"%s\". If retain is " +
+            "selected then all data in remote will be kept post-disablement and will only be deleted when it breaches expiration " +
+            "thresholds. If delete is selected then the data will be made inaccessible immediately by advancing the log start offset and will be " +
+            "deleted asynchronously.", REMOTE_LOG_DISABLE_POLICY_RETAIN, REMOTE_LOG_DISABLE_POLICY_DELETE);
```

Comment:
nit: 
1. "If _retain_ is selected" <-- the "retain" should be replaced with  `REMOTE_LOG_DISABLE_POLICY_RETAIN`
2. "If _delete_ is selected" <-- the "delete" should be replaced with  `REMOTE_LOG_DISABLE_POLICY_DELETE`